examples/08_refetch/src/fetchFuncs.ts

```diff
@@ -0,0 +1,25 @@
+import { createFetchStore } from 'react-suspense-fetch';
+
+export type UserData = {
+  data: {
+    first_name: string;
+  };
+};
+
+const fetchUser = async (userId: string) => {
+  const res = await fetch(`https://reqres.in/api/users/${userId}?delay=3`);
+  const data = await res.json();
+  return data as UserData;
+};
+
+export const store = createFetchStore(fetchUser);
+
+export const mutate = (userId: string, first_name: string) => {
+  return fetch(`https://reqres.in/api/users/${userId}?delay=3`, {
+    method: 'PUT',
+    body: JSON.stringify({ first_name }),
+  }).then((res)) => {
+    store.refetch(userId)
```

Comment:
Hm, you evict the item here. So, it's now double evicting?

Reply:
I meant to move the refetch out of the view so we refetch after every mutation.

Reply:
Ah, that makes sense.